hvplot/util.py

```diff
@@ -252,13 +260,12 @@ def process_crs(crs):
     errors = []
     if isinstance(crs, str) and crs.lower().startswith('epsg'):
         try:
-            crs = crs[5:].lstrip().rstrip()
-            return ccrs.epsg(crs)
+            crs = pyproj.CRS.from_epsg(crs).to_wkt()
         except Exception as e:
             errors.append(e)
     if isinstance(crs, int):
         try:
-            return ccrs.epsg(crs)
+            crs = pyproj.CRS.from_epsg(crs).to_wkt()
```

Comment:
Is this really needed? I would expect an integer to either fail or pass without converting it to_wkt.

Reply:
```
import cartopy.crs as ccrs

ccrs.epsg(4326)
```

Reply:
```
import pyproj
import cartopy.crs as ccrs
from hvplot.util import process_crs

process_crs(pyproj.CRS.from_epsg(4326).to_wkt())
```

Reply:
The logic around it is set up so it is okay for a function to fail. So the question is, does `process_crs(4326)` work both before and after this PR?

Reply:
Before, yes. However, due to this being removed for having incorrect assumptions, no.
```
    if proj_is_latlong(proj):
        return ccrs.PlateCarree()
```

Also to add, ccrs.epsg(3857) vs ccrs.epsg("3857") on main yields different results, which can be bad.
<img width="528" alt="image" src="https://github.com/holoviz/hvplot/assets/15331990/5121d34a-5bf5-44c1-bf99-7fe84692e878">

Reply:
This branch unifies the string/int input:

<img width="338" alt="image" src="https://github.com/holoviz/hvplot/assets/15331990/f957fd28-9ed6-48b3-a15b-a2bfd14e6380">

Reply:
Even though the projections are different types, can't they represent the same projection?

![image](https://github.com/holoviz/hvplot/assets/19758978/23fd4818-7f19-46f9-ba5f-d1a49e64482b)

https://epsg.io/3857

If you feel strongly about always using `pyproj.CRS.from_epsg(xxx).to_wkt()`. I don't have any objection, but think we could combine some of the if statements.

Reply:
Okay, I'll try to simplify the if statements.